app/assets/stylesheets/blacklight/_facets.scss

```diff
@@ -148,6 +148,22 @@ ul.facet_extended_list, .facet_extended_list ul
   @extend .clearfix;
 }
 
+ 
+// Facet modal alphabetical filter
+.alpha-filter {
+  margin-bottom: 18px;
+  text-align: center;
+
+  a {
```

Comment:
Same comment about this padding. Maybe there's a good bootstrap way to achieve the same effect?